doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -382,94 +382,64 @@ approaches that can be taken, with :numref:`Comparison of example Kubernetes net
 differences between networking solutions that consist of these options. It is important to note that different
 networking solutions require different descriptors from the Kubernetes workloads (specifically, the deployment
 artefacts such as YAML files, etc.), therefore the networking solution should be agreed between the CNF vendors and the
-CNF operators:
+CNF operators. To allow easy integration of CNF-s and the platforms it is recommended to use either the plain CNI API
+resources or the API resources defined in the v1.2 of the `Kubernetes Network Custom Resource Definition De-facto
+Standard <https://github.com/k8snetworkplumbingwg/multi-net-spec/tree/master/v1.2>`__.
 
 -  The **Default CNI Plugin** through the use of deployment specific configuration (e.g. `Tungsten Fabric
    <https://tungstenfabric.github.io/website/Tungsten-Fabric-Architecture.html#vrouter-deployment-options>`__)
 -  A **multiplexer/meta-plugin** that integrates with the Kubernetes control plane
    via CNI (Container Network Interface) and allows for use of multiple CNI plugins
    in order to provide this specific connectivity that the default Network Plugin may
```

Comment:
what specific connectivity?
what is the "the default Network Plugin"?

Reply:
The default Network Plugin is the default CNI of the cluster. 
The specific connectivity is to have multiple network connections to a Pod.  Should I try to rephrase this?

Reply:
@CsatariGergely I think you fixed it with the new wording. Thanks